### PR TITLE
Update go to v1.19

### PR DESCRIPTION
**Pull request by aledbf**
## How to test
- Open a workplace using this pull request
- Run `go version`
- Check the returned value is `go version go1.19 linux/amd64`

## Release Notes
```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

**Comment by roboquat** [comment]
@aledbf: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: .gitpod.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:af-install-evans-in-base-image.1
+image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-go.18
```
Comment (sagor999) [inline .gitpod.yml:1]: image says go.18... :smile:

**Review by sagor999 — APPROVED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (aledbf) [reply .gitpod.yml:1]: that's the number of the build  ¯\_(ツ)_/¯

**Review by jenting — APPROVED** [review]

**Review by mustard-mh — APPROVED** [review]

**Review by felladrin — APPROVED** [review]

**Review by andreafalzetti — APPROVED** [review]

**Review by Pothulapati — APPROVED** [review]
:shipit:

**Pull request merged**